content/guides/core-concepts/introduction-to-cypress.md

```diff
@@ -515,9 +527,19 @@ is what we mean when we say Cypress commands are asynchronous.
 
 #### Take this short test, for example:
 
+:::visit-mount-test-example
```

Comment:
People shouldn't be using `cy.url()` in component tests. How about this as an example?

```js
it('hides the thing when it is clicked', () => {
  cy.visit('/my/resource/path') // Nothing is actually happening yet

  cy.get('.hides-when-clicked') // Still nothing happening
    .should('be.visible')       // Still absolutely nothing
    .click()                    // Nope, nothing
    .should('not.be.visible')   // Definitely nothing happening yet
})

// Ok, the test function has finished executing...
// We've queued all of these commands and now
// Cypress will begin running them in order!
```

```js
it('hides the thing when it is clicked', () => {
  cy.mount(<MyComponent />)     // Nothing is actually happening yet

  cy.get('.hides-when-clicked') // Still nothing happening
    .should('be.visible')       // Still absolutely nothing
    .click()                    // Nope, nothing
    .should('not.be.visible')   // Definitely nothing happening yet
})

// Ok, the test function has finished executing...
// We've queued all of these commands and now
// Cypress will begin running them in order!
```

Reply:
This change doesn't seem to have been made